makefiles/boot/riotboot.mk

```diff
@@ -19,16 +22,16 @@ EPOCH := $(shell date +%s)
 APP_VER ?= $(EPOCH)
 
 # Final target for slot 0 with riot_hdr
-SLOT0_RIOT_BIN = $(BINDIR_APP)-slot0.$(APP_VER).riot.bin
-SLOT1_RIOT_BIN = $(BINDIR_APP)-slot1.$(APP_VER).riot.bin
+SLOT0_RIOT_BIN = $(BINDIR_RIOTBOOT)/slot0.$(APP_VER).bin
+SLOT1_RIOT_BIN = $(BINDIR_RIOTBOOT)/slot1.$(APP_VER).bin
```

Comment:
This shouldn't be `BINDIR_SUIT`, right?

Reply:
No, @bergzand suggested on matrix that suit files were better stored in a separate folder, so this split addressed that idea.